### ISSUE TITLE
registry endpoint access?

### ISSUE THREAD

**Issue by gpertea** [ChristopherWilks/snaptron#16]
Maybe I'm missing the obvious but I could not find an explicit WSI example/documentation related to accessing the registry endpoints as mentioned in #10. 

For example I could not figure out a way to list the **currently available compilations** using the web API (through https://snaptron.cs.jhu.edu), which seems like it could be a basic initial operation for an incoming user (i.e. it would be useful to know what datasets are available).

The snapcount vignette and user manual also did not seem to explicitly document such a basic query (only informing the user about the same old 4 initial compilations) , though users looking for the updated list can actually get it by inspecting that environment -- e.g.  `ls(snapcount::Compilation)` - however that's not made obvious in the documentation. 

Also, https://snaptron.cs.jhu.edu/data.html seems to have been last updated in 2016 -- so the `srav3h` compilation is not mentioned there, even though it is currently the largest (human) compilation actually available through snaptron.

**Comment by ChristopherWilks** [comment]
Hi @gpertea,

Thanks for the feedback.  

All good points, you didn't miss anything.  
Snaptron's documentation has lagged (and in some cases stagnated) the service itself.

I have just updated the data page to include the latest recount3-based compilations as well as a note about the registry:

https://snaptron.cs.jhu.edu/data.html

I have also trimmed the compilation list to remove deprecated compilations (e.g. `nim`, `srav1`).
